tests/test_filelock.py

```diff
@@ -333,3 +334,14 @@ def test_cleanup_soft_lock(tmp_path):
     with lock:
         assert lock_path.exists()
     assert not lock_path.exists()
+
+
+def test_logger_name():
+    # need to check both that the filelock logger exists and is not created as
```

Comment:
Please align with 120 characters width 👍 but IMHO anything this tests does is already tested above and therefore can be removed 👍

Reply:
@gaborbernat Wow that speedy review. Sure, I wasn't sure if we wanted an explicit test to test the log name rather than grouping it with the `test_simple`. I can remove it 👍